home/javascript/react/components/curate-construct/DomainInfo.ts

```diff
@@ -0,0 +1,10 @@
+function backendBaseUrl() {
+    //TODO: This is a hack to get the domain for developing locally.  It should be removed when this is deployed to production.
```

Comment:
I am not sure what this means. Would you remove this code after this sprint is deployed to prod?